mobile/android/app/build.gradle

```diff
@@ -138,6 +138,7 @@ android {
 }
 
 dependencies {
+    compile project(':react-native-gesture-handler')
```

Comment:
should also be implementation?

Reply:
Done. da300fe810d9dcb49a4a22fe087eaf20c2040dd9